OpenRA.Game/FileSystem/PackageEntry.cs

```diff
@@ -65,14 +65,14 @@ public static uint HashFilename(string name, PackageHashType type)
 						if (name.Length % 4 != 0)
 							name = name.PadRight(name.Length + (4 - name.Length % 4), '\0');
 
-						var ms = new MemoryStream(Encoding.ASCII.GetBytes(name));
-						var reader = new BinaryReader(ms);
-
-						var len = name.Length >> 2;
 						uint result = 0;
-
-						while (len-- != 0)
-							result = ((result << 1) | (result >> 31)) + reader.ReadUInt32();
+						using (var ms = new MemoryStream(Encoding.ASCII.GetBytes(name)))
+						using (var reader = new BinaryReader(ms))
```

Comment:
You can remove this one. https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/StreamExts.cs will take care of it.